g1ant-language/triggering/mail-trigger.md

```diff
@@ -2,51 +2,51 @@
 
 Mail Trigger provides information about incoming emails.
 
-**Initial Arguments**
+## Initial Arguments
 
 | Trigger Argument | Required | Default value | Description |
-| -------- | ---- | -------- | ------------- | ----------- |
-| `Host` | yes | - | imap server address |
-| `Login` | yes | - | login (e.g. email address) from which there will be sent emails |
-| `Password` | yes | - | password for this email address |
-| `Frequency` | no | 30000 | time set between updating information about incoming emails (in milliseconds) |
-| `Port` | yes | - | incoming mail server port |
+| -------- | ---- | -------- | ------------- |
+| `Host` | yes | - | IMAP server address |
+| `Login` | yes | - | Login (e.g. email address) for the account the emails will be sent from |
+| `Password` | yes | - | Password for this email account |
+| `Frequency` | no | 30000 | Time interval between updates on incoming emails (in milliseconds) |
+| `Port` | yes | - | Incoming mail server port |
 
-**Task Arguments**
+## Task Arguments
 
-Arguments generated while executing the script for each of the incoming email.
+Arguments generated while executing the script for each incoming email:
 
 | Trigger Argument | Description |
-| -------- | ---- | -------- | ------------- | ----------- |
-| `Title` | title of an incoming mail |
-| `AttachmentNames` | list of attachments in an incoming mail |
-| `Content` | incoming mail content (text) |
-| `From` | sender email address |
-| `To` | recipient email address |
-| `HtmlBody` | incoming email content (html) |
-| `Uid` | unique identifier of a mail given by a server |
-| `Date` | date sent |
-| `Priority` | priority set by a sender |
-
-**Example of defining File Trigger in Settings**
+| -------- | ---- |
+| `Title` | The title of an incoming mail |
+| `AttachmentNames` | A list of attachments in an incoming mail |
+| `Content` | Incoming mail content (text) |
+| `From` | Sender's email address |
+| `To` | Recipient's email address |
+| `HtmlBody` | Incoming mail content (html) |
+| `Uid` | A unique identifier of a mail given by a server |
+| `Date` | Sent date |
```

Comment:
date sent

Reply:
checked against reality: it's like that :D